library/common.h

```diff
@@ -57,4 +57,91 @@
  */
 #define MBEDTLS_ALLOW_PRIVATE_ACCESS
 
+/** Byte Reading Macros
+ *
+ * To tidy up code and save horizontal and vertical space, use byte
+ * reading macros to cast
```

Comment:
Please write a proper documentation for all macros including a description of the parameters. See tests/include/test/macros.h for examples.

Reply:
done, but I'm not totally sure if my documentation comments are correct/true

Reply:
They will also change as I update the GET macros like you suggested, so they will change a little bit in the next commit